src/modules/securitybaseline/src/lib/CMakeLists.txt

```diff
@@ -3,11 +3,17 @@
 
 project(securitybaselinelib)
 
-add_library(securitybaselinelib STATIC SecurityBaseline.c)
+add_library(securitybaselinelib STATIC 
+    SecurityBaseline.c)
+
 set_property(TARGET securitybaselinelib PROPERTY POSITION_INDEPENDENT_CODE ON)
 
-target_link_libraries(securitybaselinelib PRIVATE logging commonutils parsonlib)
-target_include_directories(securitybaselinelib
-    PUBLIC
-        ${MODULES_INC_DIR}
-        ${CMAKE_CURRENT_SOURCE_DIR})
\ No newline at end of file
+target_link_libraries(securitybaselinelib PRIVATE 
+    logging 
```

Comment:
can make `logging` + `commonutils` `PUBLIC`? You can remove them from your so\CMakeLists.txt `target_link_libraries`

Reply:
understood, but `PUBLIC` in this context simply means that those dependencies are transitive, so upstream targets (so's) inherit those targets. If your redefining those targets for the so then they should be made PUBLIC in the downstream target

Reply:
That is not the case here. Why would you suggest PUBLIC at all?